action/protocol/staking/ethabi/common/candidatebyaddress.go

```diff
@@ -76,7 +76,7 @@ func (r *CandidateByAddressStateContext) EncodeToEth(resp *iotexapi.ReadStateRes
 
 	data, err := r.Method.Outputs.Pack(cand)
 	if err != nil {
-		return "", nil
+		return "", err
```

Comment:
looks like this is a typo/bug in current code? we don't have a test for this?

Reply:
yeah, existed typo, but it's not easy to cause error for abi pack